.slim-lint.yml

```diff
@@ -16,6 +16,7 @@ linters:
       - Layout/IndentArray
       - Layout/IndentationConsistency
       - Layout/IndentationWidth
+      - Layout/IndentHash
```

Comment:
Slim-lint has a known issue with this Rubocop linter and they recommend it should be disabled.